content/en/docs/reference/config/proxy_extensions/stats/index.html

```diff
@@ -24,7 +24,6 @@ <h2 id="MetricConfig">MetricConfig</h2>
 <th>Field</th>
 <th>Type</th>
```

Comment:
@richardwxn can you check why this is `proxy_extensions` and why others like attribute_gen are not?
I would like all proxy_exntensions to be under `reference/config/proxy_extensions`

Reply:
the location tag of newly added proto docs are different: https://github.com/istio/proxy/blob/master/extensions/attributegen/config.proto#L21 I can fix them

Reply:
the accesslog one is correct, so we just need to update the attributegen one

Reply:
@ericvn Would you mind rerun it again after I update the proxy doc

Reply:
I can do that. Note that we are pulling from the release-1.6 branch so will need to wait on that.

Reply:
Yep I created a PR for 1.6: https://github.com/istio/proxy/pull/2833

Reply:
will let you know when it is merged